src/components/ListItemGroup.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ListItem } from './ListItem';
+import './ListItemGroup.css';
+
+export function ListItemGroup({ listItems, listToken, urgencyCategory }) {
+	const groupUrgency = listItems[0].isChecked
+		? 'Purchased'
+		: urgencyCategory[listItems[0].urgencyCategory];
+
+	return (
+		<ul>
+			<hr />
+			<h3 className="Category">{groupUrgency}</h3>
+			{listItems.map((item, i) => (
+				<ListItem
+					key={item.name + i}
+					{...item}
+					item={item}
+					listToken={listToken}
+					urgencyCategory={urgencyCategory}
+				/>
+			))}
```

Comment:
What happens if one of the categories has no items? Can we have a placeholder here to indicate to the user that "yes, there are no items here". Without some sort of placeholder message, the user might not sure if it was a loading error, or if there really aren't any in that category currently.

Reply:
Great point. I added a placeholder and included the number of items in each category, so if there are no items in 'Soon', the user will see 'Soon (0)'